docs/features/series.md

```diff
@@ -0,0 +1,205 @@
+[Shelves and libraries](shelves-and-libraries.md) load their data using the [GET playlist endpoint](https://developer.jwplayer.com/jwplayer/reference/get_v2-playlists-playlist-id). Some items in this  playlis refer to series. These are identified using the `seriesId` , which links to the  playlist that contains the episodes. 
+   
+   ```
+   GET playlist/o45EkQBf
+   {
+   "title":"All Courses",
+   "description":"",
+   "kind":"DYNAMIC",
+   "feedid":"o45EkQBf",
+   "playlist":[
+    {
+      "mediaid":"txHnOU0x",
+      "seriesId":"xdAqW8ya",
+      "episodes":"4"
+      "title":"Primitive Animals",
+      "image":"https://content.jwplatform.com/v2/media/txHnOU0x/poster.jpg?width=720",
+      "duration":0,
+      "pubdate":1615379806,
+      "description":"Aimed at beginners, this workshop helps you connect basic Blender functionality into a complete workflow and mindset for building characters.",
+      "tags":"course,Beginner"
+    },
+    { }
+   ]
+   }
+   ```
+   
+### Serie detail window
+The serie detail window loads the series playlist using the [GET playlist endpoint](https://developer.jwplayer.com/jwplayer/reference/get_v2-playlists-playlist-id). The episodelabel(e.g. `S1:E1`)  is coming from `seasonNumber` and `episodeNumber` 
+   
+   ```
+   GET playlist/xdAqW8ya
+   {
+   "title":"Primitive Animals",
+   "description":"Aimed at beginners, this workshop helps you connect basic Blender functionality into a complete workflow and mindset for building characters.",
+   "kind":"DYNAMIC",
+   "feedid":"xdAqW8ya",
+   "playlist":[
+    {
+      "title":"Blocking",
+      "mediaid":"zKT3MFut",
+      "image":"https://content.jwplatform.com/v2/media/zKT3MFut/poster.jpg?width=720",
+      "feedid":"xdAqW8ya",
+      "duration":345,
+      "pubdate":1615370400,
+      "description":"If you're brand new to Blender or are getting stuck, check out the Blender 2.8 Fundamentals series.",
+      "tags":"seriesId_xdAqW8ya,lesson",
+      "seasonNumber":"1",
+      "episodeNumber":"1",
+      "genre":"Beginner",
+      "rating":"CC-BY"
+    },
+    {
+      "title":"Modeling",
+      "mediaid":"QLPqP2u8",
+      "image":"https://content.jwplatform.com/v2/media/QLPqP2u8/poster.jpg?width=720",
+      "feedid":"xdAqW8ya",
+      "duration":184,
+      "pubdate":1615374000,
+      "description":"If you're brand new to Blender or are getting stuck, check out the Blender 2.8 Fundamentals series.",
+      "tags":"seriesId_xdAqW8ya,lesson",
+      "seasonNumber":"1",
+      "episodeNumber":"2",
+      "genre":"Beginner",
+      "rating":"CC-BY"
+    }
+   }
+   ```
+
+## Native series - Coming soon
+JW Player will get native series management from the JW Dashboard:
+- simplifies the series creation workflow
+- automatically calculate the number of episodes and duration of series
+- contains trailers and bonus content
+  This section describes how this will work. 
+  
+### Creating native series in the dashboard
+1. Customers define series 
```

Comment:
Again, I think it's risky to put too much detail about an external application here and it getting out of date.

Reply:
I would like to leave this one here, until we have end-user documentation for the new series mgmt.

We anyway need to revisit this documentation when the new series get introduced.